doc/api/process.markdown

```diff
@@ -169,15 +185,18 @@ event tells you when the list of unhandled rejections shrinks.
 For example using the rejection detection hooks in order to keep a list of all
 the rejected promises at a given time:
 
-    var unhandledRejections = [];
+    var unhandledRejections = new Set();
     process.on('unhandledRejection', function(reason, p) {
-        unhandledRejections.push(p);
+      unhandledRejections.add(p);
     });
     process.on('rejectionHandled', function(p) {
-        var index = unhandledRejections.indexOf(p);
-        unhandledRejections.splice(index, 1);
+      unhandledRejections.delete(p);
     });
 
+You could then record this list in some error log, either periodically or upon
+process exit. This would avoid the false positive seen in the above resource
```

Comment:
That's not a useful definition for false positive.

This code path doesn't have any bugs regarding to not handling errors, and it would cause the server logs to be filled with false positives regardless if you use periodic logging or logging-on-exit:

```js
function doIt(request) {
  var a = Promise.reject(new Error());
  setTimeout(function() {
    a.catch(function(e) {
      request.sendError(e.toString());
    });
  }, 10);
}

server.onRequest(doIt);
```